apps/a11y-tests/scripts/generate-sarif-reports.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
```

Comment:
Nitpick but I recommend adding `// @ts-check` to the top of all new JS files. That way you get some of the benefits of type checking at least within VS Code.

Reply:
Added for all JS code. Thanks!